src/app/settings/organization-plans.component.ts

```diff
@@ -159,6 +159,16 @@ export class OrganizationPlansComponent implements OnInit {
         return subTotal;
     }
 
+    get taxCharges() {
+        return this.taxComponent != null && this.taxComponent.taxRate != null ?
```

Comment:
Did this pass the linter? (i.e., `this.taxComponent !== null`)

Reply:
Yes, the linter is happy with it. I left it this way to still cautiously check for any undefineds, but that isn't really needed. Want to change to !==?